apps/alphabet_mobile/SETUP_README.md

```diff
@@ -0,0 +1,65 @@
+# Create a React Native project on Ubuntu 22.04
```

Comment:
You can call this `SETUP.md` and link it to `README.md` under a heading
# Setup
...